### PR TITLE
SCRUM-12 - Feature/scrum 12/spinner

### PR DESCRIPTION
**Pull request by Annel18**

### CODE REVIEW

File: src/utils/loaders/getCityFeed.ts [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+  let location: string;
+  if (typeof input === 'string') {
+    location = input;
+  } else {
+    const { request, params } = input;
+    const url = new URL(request.url);
+    location = params.location || url.searchParams.get('location') || 'defaultLocation';
+  }
+
+  const response = await axios.get(`https://api.waqi.info/feed/${location}/?token=${token}`);
```
Comment (rasimsen) [inline src/utils/loaders/getCityFeed.ts:16]: better to get them from Constant.ts file (URL)

Comment (rasimsen) [inline src/utils/loaders/getCityFeed.ts:16]: can you create your own http methods file , such as : GET, PUT, POST, DELETE. and use axis only in there and in a ages, you can use only your own HTTP methods, so later on, you can easily replace axios with something else.

Comment (Annel18) [reply src/utils/loaders/getCityFeed.ts:16]: constants.ts file added

Comment (Annel18) [reply src/utils/loaders/getCityFeed.ts]: http method file created with axios CRUD

**Pull request merged**